tests/orders/tests-edd-payment.php

```diff
@@ -911,6 +911,26 @@ public function test_get_tax_rate_returns_decimal() {
 		$this->assertEquals( 0.2, $payment->tax_rate );
 	}
 
+	public function test_receipt_show_download_files() {
```

Comment:
```suggestion
	/**
	 * This tests backwards compatibility in `edd_receipt_show_download_files()` when passing
	 * an array as the third parameter instead of the new `Order_Item` object. This test
	 * ensures that the function successfully converts that array to an order item object
	 * to be passed through to the filter `edd_order_receipt_show_download_files`.
	 *
	 * @covers \edd_receipt_show_download_files()
	 */
	public function test_receipt_show_download_files() {
```

Just to help remember what the test is for and exactly what it's covering.